meson.build

```diff
@@ -347,6 +354,7 @@ cdata.set_quoted(
 cdata.set_quoted('G_LOG_DOMAIN', 'flatpak')
 cdata.set_quoted('GETTEXT_PACKAGE', 'flatpak')
 cdata.set('FUSE_USE_VERSION', fuse_api)
+cdata.set_quoted('FUSERMOUNT', fusermount.path())
```

Comment:
Isn't this going to fail if `fusermount` was set to a string, for example `-Dsystem_fusermount=/usr/bin/fusermount`? You're mixing types for the `fusermount` variable: sometimes it's a string, but sometimes it's a program.

I think what we need is logic like this (untested):

```
fusermount = get_option('fusermount')

if fusermount == ''
  if fuse_api >= 30
    fusermount_program = find_program('fusermount3', required: true)
  else
    fusermount_program = find_program('fusermount', required: true)
  endif
  if meson.version().version_compare('>=0.55')
    fusermount = fusermount_program.full_path()
  else
    fusermount = fusermount_program.path()
  endif
endif
```

and then use `fusermount` everywhere else.